### ISSUE TITLE
Selection bug when DST in Australia in Safari Mac

### ISSUE THREAD

**Issue by arshaw** [fullcalendar/fullcalendar#2396]
Originally reported on Google Code with ID 2131
```
Issue: 
Click on a selectable time slot in day view on specific date/time of 5th April 5pm-5:30pm,
will incorrectly highlight cell 6pm-6:30pm (skips ahead 1 hour). 

Must be using MAC Safari web browser, and set timezone to Sydney, Australia (Australia
eastern Daylight saving time).

To Reproduce:
1) Change timezone to Sydney, Australia (Australia eastern Daylight saving time) if
not already. (on OSX - System preferences > Date & Time > Time Zone Tab > Uncheck Set
Timezone Automatically using current location > select Sydney, Australia from world
map. (Location _must_ be Sydney, Australia it cannot be any other location because
some other locations do not use daylight saving)

2) Load safari with url - http://jsfiddle.net/7PgLM/1/

3) Click "day", change date to April 5th, then click time slot 5pm. The actual row
selected will be incorrect and skip ahead 1 hour.

4) Also note, the popup alert within dayClick() callback is displaying a matching incorrect
date.

NOTE: 
- Date of 5th April is important as its change over of daylight saving
- Must be using Mac Safari browser, safari on PC does not have bug, other browsers
on mac do not have bug. (tested chrome, firefox)
- Same problem using safari iPad and iPhone
- Location _must_ be Sydney, Australia it cannot be any other location because some
other locations do not use daylight saving

```

Reported by `robert1000` on 2014-04-10 05:43:22

Imported with **2** stars.

**Comment by pgbrown** [comment]
I wonder if anyone has a solution on how to fix this in version 1.5 of the calendar with code. I've tried to work it out to no avail. At the moment we are detecting the browser and putting up a message to suggest users try another browser for this period.

**Comment by fonini** [comment]
This bug is related to Daylight Saving Time. Here in Brazil (America/Sao_Paulo), it goes from 2015-10-18 to 2016-02-21.

**Comment by dhara15101991** [comment]
i have find that

dayClick: function(date, allDay, jsEvent, view)
{
       alert(date); // Gives Sat Nov 21 2015 19:00:00 GMT+0000
}
Here date gives me Sat Nov 21 2015 19:00:00 GMT+0000. And my timezone is Asia/kolkata (GMT+05:30). so it automatically add 05:30 hours to 19:00:00. So date is converted to next day. 

so i have to calculate timezone with date given by fullcalender 
dayClick: function(date, allDay, jsEvent, view)
    {

           var tzDifference = date.getTimezoneOffset() //this gives me timezone difference of local and UTC time in minutes
           var offsetTime = new Date(date.getTime() + tzDifference * 60 * 1000); //this will calculate time in point of view local time and set date
    }
I have set 
    ignoreTimezone: false, 
    timezone:'Asia/Kolkata',

But i doesn't consider in date so it gives me Sat Nov 21 2015 19:00:00 GMT+0000. So i calculate the date and get it proper. May be this is problem but this works for me.

**Comment by arshaw** [comment]
this has been fixed in a beta release. please try it out!
https://github.com/fullcalendar/fullcalendar/releases/tag/v2.5.0-beta2

**Comment by arshaw** [comment]
fixed in [v2.5.0](https://github.com/fullcalendar/fullcalendar/releases/tag/v2.5.0)
